liquid.gemspec

```
@@ -4,11 +4,11 @@
 lib = File.expand_path('../lib/', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
-require "liquid/version"
+require "liquid5/version"
 
 Gem::Specification.new do |s|
   s.name        = "liquid"
```

this will not be installable in tandem with v3 if it keeps the same gem name

I think this file needs to change its name to `liquid5.gemspec` and this:

```suggestion
  s.name        = "liquid5"
```